gst-plugin-pravega/src/pravegasink/imp.rs

```diff
@@ -281,6 +375,31 @@ impl ObjectImpl for PravegaSink {
                 None,
                 glib::ParamFlags::WRITABLE,
             ),
+            glib::ParamSpec::new_string(
```

Comment:
You should use `glib::ParamSpec::new_enum` for retention type.

Reply:
done